### ISSUE TITLE
Rename "Quantum" to "Windows"

### ISSUE THREAD

**Issue by davehunt** [mozilla-frontend-infra/firefox-health-dashboard#575]
Let's rename "Quantum" to "Windows" as the latter is more accurate in describing the data that's being plotted on these pages. We should also redirect "quantum" URLs to "windows" so that we don't break any bookmarks.

**Comment by Rokiis** [comment]
Hello, is it still available? Would like to contribute.

Thanks.

**Comment by Rokiis** [comment]
Quantum Changed to Windows, All tests pass, test -u all pass, filenames updated as well as classnames containing works windows. 

New branch created Quantum-to-Windows  

However, I can't push to the repo

| λ git push origin Quantum-to-Windows                                                                                                     
| remote: Permission to mozilla-frontend-infra/firefox-health-dashboard.git denied to Rokiis.                                              
| fatal: unable to access 'https://github.com/mozilla-frontend-infra/firefox-health-dashboard.git/': The requested URL returned error: 403 |


Any help?

**Comment by Rokiis** [comment]
PR #611 sent for review.

**Comment by klahnakoski** [comment]
@Rokiis you must

1. fork this repo to your Github account, then 
2. clone that fork to your local machine
3. Make your changes, and push to your Github account
4. Make a PR here

**Comment by Rokiis** [comment]
Yeah, that's how I've managed to push the PR through. Could you please review?

**Comment by divyanshu-rawat** [comment]
@klahnakoski is this open to work on?

**Comment by Rokiis** [comment]
@divyanshu-rawat Im still pending code review

**Comment by shreygupta2809** [comment]
Hi,Is this bug still available for fixing. I would like to work on it.
Thanks
